lib/xboxkrnl/xboxkrnl.h

```diff
@@ -2861,6 +2861,7 @@ XBAPI NTSTATUS NTAPI NtOpenDirectoryObject
 
 #define FILE_DEVICE_FILE_SYSTEM 0x00000009
 #define FSCTL_DISMOUNT_VOLUME CTL_CODE(FILE_DEVICE_FILE_SYSTEM, 8, METHOD_BUFFERED, FILE_ANY_ACCESS)
+#define FSCTL_GET_OBJECT_ID CTL_CODE(FILE_DEVICE_FILE_SYSTEM, 39, METHOD_BUFFERED, FILE_ANY_ACCESS)
```

Comment:
This appears to be unused? Why is this part of this PR?

Reply:
Removed along with the other objectid checking code.